dcat/rdf/dcat.ttl

```diff
@@ -174,9 +174,9 @@ dcat:Catalog
       owl:allValuesFrom dcat:Resource ;
       owl:onProperty dct:hasPart ;
     ] ;
-  skos:definition "A curated collection of metadata about datasets and data services."@en ;
-  skos:definition "Una colección seleccionada de metadatos sobre conjuntos de datos o servicios de datos."@es ;
-  skos:definition "Una raccolta curata di metadati sui dataset e relativi servizi."@it ;
+  skos:definition "A curated collection of metadata about cataloged resources (e.g., datasets and data services in the context of a data catalog)."@en ;
+  skos:definition "Una colección curada de metadatos sobre recursos catalogados (por ejemplo, conjuntos de datos y servicios de datos en el contexto de un catálogo de datos)."@es ;
+  skos:definition "Una raccolta curata di metadati sulle risorse catalogate (ad es. sui dataset e relativi servizi nel contesto di cataloghi di dati)."@it ;
   skos:definition "Une collection élaborée de métadonnées sur les jeux de données."@fr ;
   skos:definition "Řízená kolekce metadat o datových sadách a datových službách."@cs ;
```

Comment:
@jakubklimek  would be the following an acceptable translation for ""A curated collection of metadata about cataloged resources (e.g., datasets and data services in the context of a data catalog)."?

```suggestion
  skos:definition "Řízená kolekce metadat o katalogizovaných zdrojích (např.  datových sadách a datových službách v souvislosti s katalogem dat)"@cs ;
```